v2/fleet/mesos-slave.service

```diff
@@ -33,7 +31,6 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
     --containerizers=docker,mesos \
     --executor_registration_timeout=10mins \
     --hostname=`curl -s http://169.254.169.254/latest/meta-data/public-hostname` \
-    --isolation=cgroups/cpu,cgroups/mem \
```

Comment:
in our config, it doesn't seem to do anything. This was a recommendation from somewhere to get mesos-in-a-container to perform better. But It's not the default and seems to work fine without it, so  I'm removing it.

From the (sparse) documentation:
```
Isolation mechanisms to use, e.g., 'posix/cpu,posix/mem', or 'cgroups/cpu,cgroups/mem', or network/port_mapping (configure with flag: --with-network-isolator to enable), or 'external', or load an alternate isolator module using the --modules flag. Note that this flag is only relevant for the Mesos Containerizer. (default: posix/cpu,posix/mem)
```